sdk/cosmosdb/cosmos/src/queryExecutionContext/orderByComparator.ts

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { NonStreamingOrderByResult } from "./nonStreamingOrderByResult";
+
+/** @hidden */
+const TYPEORDCOMPARATOR: {
+  [type: string]: { ord: number; compFunc?: (a: any, b: any) => number };
+} = Object.freeze({
+  NoValue: {
+    ord: 0,
+  },
+  undefined: {
```

Comment:
Isn't NoValue used for undefined values?

Reply:
Yes, this is equivalent to null and undefined both. This is currently kept same as `orderByDocumentProducer` class to keep the behavior same.